src/swarm-tui/Swarm/TUI/View.hs

```diff
@@ -1276,15 +1276,31 @@ explainCapabilities gs e
       ]
 
   renderCmdInfo c =
-    padTop (Pad 1) $
-      vBox
-        [ hBox
-            [ padRight (Pad 1) (txt . syntax $ constInfo c)
-            , padRight (Pad 1) (txt ":")
-            , withAttr magentaAttr . txt . prettyText $ inferConst c
-            ]
-        , padTop (Pad 1) . padLeft (Pad 1) . txtWrap . briefDoc . constDoc $ constInfo c
-        ]
+    Widget Fixed Fixed $ do
+      ctx <- getContext
+      let w = ctx ^. availWidthL
+          constType = inferConst c
+          info = constInfo c
+          requiredWidthForTypes = textWidth (syntax info <> " : " <> prettyTextLine constType)
```

Comment:
Today I learned about `textWidth`!  I didn't know about this before.  There are probably some other places in the codebase where we ought to be using it instead of `length`!  https://hackage.haskell.org/package/brick-2.3.1/docs/Brick-Widgets-Core.html#t:TextWidth , for anyone else reading this.

Reply:
@byorgey it will be more noticeable with Unicode and different terminals; see:
https://github.com/jtdaugherty/vty?tab=readme-ov-file#multi-column-character-support